src/utils/transclude-directive.js

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
I'm struggling on trying to create directives on the fly in the test spec which is what I need to do in order to test the scope and DOM structures based on the transclusion type. The example in ng-doc show this and tests this pretty well.

I'd like to defer this. We can add an issue and I will address when I can. I need to get a 2.1 label on angulary-patternfly for today as there are folks that need the latest today and I'd like for the data list to be included.